<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FTransactionGiftEdit</class>
 <widget class="QDialog" name="FTransactionGiftEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date and time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateTimeEdit" name="dtDateTime">
     <property name="currentSection">
      <enum>QDateTimeEdit::DaySection</enum>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy HH:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Thing</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ui::selector::WThingSelectButton" name="pwThingSelector" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ui::selector::WValuableSelectButton" name="pwValuableSelector" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>From</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>To</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="ui::selector::WContractorSelectButton" name="pwAccountFrom" native="true">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ui::selector::WContractorSelectButton" name="pwAccountTo" native="true">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ui::selector::WThingSelectButton</class>
   <extends>QWidget</extends>
   <header>wthingselectbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ui::selector::WValuableSelectButton</class>
   <extends>QWidget</extends>
   <header>wvaluableselectbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ui::selector::WContractorSelectButton</class>
   <extends>QWidget</extends>
   <header>wcontractorselectbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FTransactionGiftEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FTransactionGiftEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
